### PR TITLE
Bethesda.Launcher version 1.75.0

### PR DESCRIPTION
**Pull request by mddtest**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/15829)

### CODE REVIEW

File: manifests/b/BethesdaSoftworks/Bethesda/1.75.0.0/BethesdaSoftworks.Bethesda.locale.en-US.yaml [hunk 1]
```diff
@@ -5,6 +5,7 @@ PackageVersion: 1.75.0.0
 PackageLocale: en-US
 Publisher: Bethesda Softworks
 PackageName: Bethesda.net Launcher
+Url: https://bethesda.net/en/game/bethesda-launcher
```
Comment (ItzLevvie) [inline manifests/b/BethesdaSoftworks/Bethesda/1.75.0.0/BethesdaSoftworks.Bethesda.locale.en-US.yaml:8]:
```suggestion
PackageUrl: https://bethesda.net/en/game/bethesda-launcher
```

Comment (ItzLevvie) [reply manifests/b/BethesdaSoftworks/Bethesda/1.75.0.0/BethesdaSoftworks.Bethesda.locale.en-US.yaml:8]: You also need to change `PackageVersion: 1.75.0.0` to `PackageVersion: 1.75.0` (and update the version number folder) to match the version number it shows in Control Panel.
image

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
You also need to change `PackageVersion: 1.75.0.0` to `PackageVersion: 1.75.0` (and update the version number folder) to match the version number it shows in Control Panel.
image

File: manifests/b/BethesdaSoftworks/Bethesda/1.75.0.0/BethesdaSoftworks.Bethesda.locale.en-US.yaml [hunk 2]
```diff
@@ -5,6 +5,7 @@ PackageVersion: 1.75.0.0
 PackageLocale: en-US
 Publisher: Bethesda Softworks
 PackageName: Bethesda.net Launcher
+Url: https://bethesda.net/en/game/bethesda-launcher
 License: Microsoft
 ShortDescription: Bethesda.net Launcher Setup
```
Comment (ItzLevvie) [inline manifests/b/BethesdaSoftworks/Bethesda/1.75.0.0/BethesdaSoftworks.Bethesda.locale.en-US.yaml:10]:
```suggestion
ShortDescription: Play the latest Bethesda PC games using the Bethesda.net Launcher.
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by KaranKad** [comment]
This application has quite wrong metadata already merged into main branch. I'm thinking of deleting this one and creating a new one.

File: manifests/b/BethesdaSoftworks/Bethesda/1.75.0.0/BethesdaSoftworks.Bethesda.locale.en-US.yaml [hunk 3]
```diff
@@ -5,6 +5,7 @@ PackageVersion: 1.75.0.0
 PackageLocale: en-US
 Publisher: Bethesda Softworks
 PackageName: Bethesda.net Launcher
+PackageUrl: https://bethesda.net/en/game/bethesda-launcher
 License: Microsoft
```
Comment (ItzLevvie) [inline manifests/b/BethesdaSoftworks/Bethesda/1.75.0.0/BethesdaSoftworks.Bethesda.locale.en-US.yaml:9]:
```suggestion
License: Copyright (c) 2021 Bethesda Softworks.
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Change the `PackageIdentifier` to `Bethesda.Launcher`; and update the folders and the file names.

File: manifests/b/BethesdaSoftworks/Bethesda/1.75.0/BethesdaSoftworks.Bethesda.locale.en-US.yaml [hunk 4]
```diff
@@ -1,11 +1,12 @@
...
-ManifestVersion: 1.0.0
+# Created using wingetcreate
+
+PackageIdentifier: BethesdaSoftworks.Bethesda
+PackageVersion: 1.75.0.0
+PackageLocale: en-US
+Publisher: Bethesda Softworks
+PackageName: Bethesda.net Launcher
+PackageUrl: https://bethesda.net/en/game/bethesda-launcher
+License: Microsoft
```
Comment (mddtest) [inline manifests/b/BethesdaSoftworks/Bethesda/1.75.0/BethesdaSoftworks.Bethesda.locale.en-US.yaml:9]: License: Copyright (c) 2021 Bethesda Softworks.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
`PackageIdentifier` needs to be changed from `BethesdaSoftworks.Bethesda` to `Bethesda.Launcher` and the folders and files names also have to be updated.

**Comment by KaranKad** [comment]
I've created a new manifest for this application

**Comment by KaranKad** [comment]
You need to edit `manifests/b/BethesdaSoftworks/Bethesda` to `manifests/b/Bethesda/Launcher`

**Review by mddtest — COMMENTED** [review]
updated

**Comment by KaranKad** [comment]
Try closing this pr and then reopening it to restart validation.

**Comment by KaranKad** [comment]
I think this extra file is causing the issue. https://github.com/microsoft/winget-pkgs/pull/15829/files#diff-83f03399d07cfae47d3ca618c4bae70efd0a36048ce440818aa1f10d21e17e98

**Comment by mddtest** [comment]
> 
> 
> Change the `PackageIdentifier` to `Bethesda.Launcher`; and update the folders and the file names.

updated per ask but will not validate now

**Comment by mddtest** [comment]
> 
> 
> I think this extra file is causing the issue. https://github.com/microsoft/winget-pkgs/pull/15829/files#diff-83f03399d07cfae47d3ca618c4bae70efd0a36048ce440818aa1f10d21e17e98

Thats what I was thinking but will not let me delete it.

**Pull request closed without merging**